recipes/newrelic/infrastructure/ohi/mysql/debian.yml

```diff
@@ -188,21 +188,20 @@ install:
 
           sudo cp /etc/newrelic-infra/integrations.d/mysql-config.yml.sample /etc/newrelic-infra/integrations.d/mysql-config.yml;
           sudo tee /etc/newrelic-infra/integrations.d/mysql-config.yml > /dev/null <<EOT
-          integration_name: com.newrelic.mysql
-
-          instances:
-            - name: mysql-status
-              command: status
-              arguments:
-                hostname: $NR_CLI_DB_HOSTNAME
-                port: $NR_CLI_DB_PORT
-                username: $NR_CLI_DB_USERNAME
-                password: $NR_CLI_DB_PASSWORD
-                database:
-                extended_metrics: true
-                extended_innodb_metrics: true
-                extended_myisam_metrics: true
-                remote_monitoring: true
+          integrations:
+            - name: nri-mysql
+              env:
+                HOSTNAME: $NR_CLI_DB_HOSTNAME
+                PORT: $NR_CLI_DB_PORT
```

Comment:
is it normal that the notation is different for vars? not {{...}}

Reply:
this is depending where it's commig from, {{.NR_CLI_DB_USERNAME} comes from the go templating engine because it was retrieved by the go task, and go task will replace it on runtime before executing the shell script,  while the $NR_CLI_DB_HOSTNAME it's the actual shell env variable retrieved inside the shell script